css-typed-om/Overview.bs

```diff
@@ -3574,6 +3574,11 @@ which will be filled in with more detail as we go.
 
 : <a>align-content</a>
 ::
+    1. If the specified or computed value is the ''normal'' keyword,
+        return the result of reifying the "normal" keyword.
+    2. If the value is a keyword from <<content-distribution>>,
```

Comment:
Is this legit, or do I need to specify every keyword?

Reply:
This is legit.